.pre-commit-config.yaml

```
@@ -34,6 +34,7 @@ repos:
     hooks:
       - id: codespell
         exclude_types: ["xml"] # ignore sample xml file
+        args: [--ignore-words-list=archiv] # ignore German "archiv" in URLs
```

```suggestion
        args: [--uri-ignore-words-list=archiv] # ignore German "archiv" in URLs
```
This seems a bit better since the only occurrence of this word is in a URL and it wouldn't be surprising if the English word "archive" might appears somewhere in this codebase.

I ran this locally and the codespell pre-commit check passes with this new configuration.